jjb/dynamic/scale-ci_install_aws.yml

```diff
@@ -17,7 +17,7 @@
 
         export ANSIBLE_FORCE_COLOR=true
         ansible --version
-        time ansible-playbook -vv -i inventory OCP-4.X/install-on-aws.yml
+        time ansible-playbook -vv -i inventory OCP-4.X/install-on-aws.yml | tee -a install.timing
```

Comment:
any reason for the `-a`?  It would seem we should ditch the `-a`, use a timestamp w/ the name `install-<timestamp>.timing` ?

Reply:
Makes sense, updated the PR. PTAL.